configure.ac

```diff
@@ -14,6 +14,7 @@ AC_PROG_CXX
 
 AC_SYS_LARGEFILE
 AC_FUNC_FSEEKO
+AX_PTHREAD
```

Comment:
Consider adding `AC_CONFIG_MACRO_DIRS([m4])` to configure.ac too.

Reply:
I am not sure if its preferable to use `AC_CONFIG_MACRO_DIRS([m4])` in `configure.ac`, `ACLOCAL_AMFLAGS = -I m4` in `Makefile.am` or both. Adding both is the only way to make `libtoolize` not complain, but all of the options seem to work fine.

I updated this PR to use both.